src/rules/pylint/settings.rs

```diff
@@ -56,11 +56,7 @@ pub struct Options {
     #[option(default = r"5", value_type = "int", example = r"max-args = 5")]
     /// Maximum number of arguments allowed for a function definition (see: `PLR0913`).
     pub max_args: Option<usize>,
-    #[option(
-        default = r"100",
-        value_type = "int",
-        example = r"max_statements = 100"
-    )]
+    #[option(default = r"50", value_type = "int", example = r"max_statements = 50")]
```

Comment:
Nit: in the example, use `max-statements`, since we use that casing in the TOML.

Reply:
Thank you